dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParams.java

```diff
@@ -139,9 +141,9 @@ public class EventRequestParams implements PageRequestParams {
 
   private Date scheduledBefore;
 
-  private Date updatedAfter;
+  private StartDate updatedAfter;
```

Comment:
These types should get an OpenAPI description. I think @jbee is changing how descriptions can be added. Maybe create a task that we add a description once his changes are through.